source/server/server.cc

```diff
@@ -506,7 +506,8 @@ void InstanceImpl::initialize(const Options& options,
 
 void InstanceImpl::onClusterManagerPrimaryInitializationComplete() {
   // If RTDS was not configured the `onRuntimeReady` callback is immediately invoked.
-  Runtime::LoaderSingleton::get().startRtdsSubscriptions([this]() { onRuntimeReady(); });
+  Runtime::LoaderSingleton::get().startRtdsSubscriptions(
+      [this](absl::string_view) { onRuntimeReady(); });
```

Comment:
One thing I notice looking at this PR is that most callbacks look like this now:
```c++
[this](absl::string_view) { /* ignore the string_view param! */ }
```
(I'm randomly leaving the comment here on this line, but it applies everywhere). This feels like a code smell; the name is only used internally in the init manager for debugging. Could there be a different way to implement this that doesn't require leaking names into unrelated code like this?

Reply:
That's what's annoying me these days too because I need to pass the target name info into the callback so that the manager could update its status.